test/test_general_workflow.py

```diff
@@ -65,6 +71,10 @@ def test_general_workflow(expected_hash):
     # check block and receipt after deployment
     assert_negative_block_input()
 
+    # check if in lite mode expected block hash is 0x1
```

Comment:
do you mean this as a TODO for later?

Reply:
No, it will be as it is. Block hash is 0x1. I just created a new task for block problem #253.

If it will be possible to fix that I'll fix that. If not I'll just leave it as it is because it will be only possible to check it in lite mode.

Reply:
So if #253 is resolved successfully, we can remove the `if`. Resolving this comment.